src/backend/base/langflow/components/brightdata/brightdata_structured_data.py

```
@@ -0,0 +1,1108 @@
+            
+            if detected_type:
+                dataset_config = self._get_all_datasets()[detected_type]
+                analysis_data["recommended_dataset_info"] = {
+                    "display_name": dataset_config["display_name"],
+                    "category": dataset_config["category"],
+                    "description": dataset_config["description"],
+                }
+            
+            return Data(data=analysis_data)
+            
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)